src/sql-parser/src/parser.rs

```diff
@@ -2629,14 +2629,22 @@ impl<'a> Parser<'a> {
         let if_not_exists = self.parse_if_not_exists()?;
         let name = self.parse_item_name()?;
 
-        // Webhook Source, which works differently than all other sources.
-        if self.peek_keywords(&[FROM, WEBHOOK]) {
-            return self.parse_create_webhook_source(name, if_not_exists);
-        }
-
         let (col_names, key_constraint) = self.parse_source_columns()?;
+
+        // For webhook sources we require `IN CLUSTER`, so track the position for an error message.
+        let cluster_err = self.expected(self.peek_pos(), "IN CLUSTER", self.peek_token());
```

Comment:
You can probably replace this with `let cluster_err = self.expect_keywords(&[IN, CLUSTER]);`, though I don't really know which is better.